README.md

```diff
@@ -36,7 +36,6 @@ A set of CSS styles and React components built with design in mind.
 - [Select](https://github.com/mongodb/leafygreen-ui/tree/main/packages/select)
 - [Side Nav](https://github.com/mongodb/leafygreen-ui/tree/main/packages/side-nav)
 - [Stepper](https://github.com/mongodb/leafygreen-ui/tree/main/packages/stepper)
-- [Syntax](https://github.com/mongodb/leafygreen-ui/tree/main/packages/syntax)
```

Comment:
Can also be removed from global tsconfig

Reply:
`build.tsconfig.json`

Reply:
Nice call